interface/patient_file/summary/rx_left.php

```diff
@@ -41,7 +41,7 @@
 </tr>
 <tr>
 <td>
-<?php if ($GLOBALS['rx_show_drug-drug']){ ?>
+<?php if ($GLOBALS['weno_rx_enable']){ ?>
```

Comment:
@juggernautsei . Guessing you are using a older version of the codebase. That global is in the current codebase:
https://github.com/openemr/openemr/search?utf8=%E2%9C%93&q=rx_show_drug-drug
(would be nice to keep this drug-drug checker features still working for the non-weno users also, if able too)